lib/builder/factory.js

```diff
@@ -74,11 +74,11 @@ factory.getExecutablePath = function (platform) {
     if (platform === 'aix') {
         ret = '/usr/sbin/ping';
     } else if (factory.isLinux(platform)) {
-        ret = '/bin/ping';
+        ret = v6 ? '/sbin/ping6' : '/bin/ping';
     } else if (factory.isWindow(platform)) {
         ret = process.env.SystemRoot + '/system32/ping.exe';
```

Comment:
Wondering whether we should support the '-6' for windows command, per docs: https://ss64.com/nt/ping.html
BTW I don't have access to to Windows machine to validate this behaviour.